engine/cdr.go

```diff
@@ -55,6 +55,7 @@ func NewCDRFromExternalCDR(extCdr *ExternalCDR, timezone string) (*CDR, error) {
 		}
 	}
 	if len(extCdr.CostDetails) != 0 {
+		cdr.CostDetails = &EventCost{}
```

Comment:
in the future you can use more elegant solution with new (ie: cdr.CostDetails = new(EventCost)